source/_patterns/03-templates/55-monthly-good/02-confirm.mustache

```diff
@@ -2,6 +2,23 @@
 	<div class="columns  monthly-good monthly-good-confirm">
 		<p>Monthly Good <strong>confirm</strong> page placeholder</p>
 
-		<p>go to <a href="{{billing_href}}">billing page</a></p>
+		{{#is_new_user}}<p>is new user</p>{{/is_new_user}}
+		{{^is_new_user}}<p>is existing user</p>{{/is_new_user}}
+		{{#has_monthly_good}}<p>already has MG</p>{{/has_monthly_good}}
+		{{^has_monthly_good}}<p>does NOT have MG</p>{{/has_monthly_good}}
+		{{#is_logged_in}}<p>is logged in</p>{{/is_logged_in}}
+		{{^is_logged_in}}<p>is NOT logged in</p>{{/is_logged_in}}
+		{{#has_legacy_subscription}}<p>has legacy subscription</p>{{/has_legacy_subscription}}
+		{{^has_legacy_subscription}}<p>NO legacy subscription</p>{{/has_legacy_subscription}}
+		{{#has_auto_deposit}}<p>has auto-deposit</p>{{/has_auto_deposit}}
+		{{^has_auto_deposit}}<p>NO auto-deposit</p>{{/has_auto_deposit}}
+		{{#has_billing_agreement}}<p>has billing agreement</p>{{/has_billing_agreement}}
+		{{^has_billing_agreement}}<p>NO billing agreement</p>{{/has_billing_agreement}}
+		{{#has_auto_lending}}<p>has auto-lending</p>{{/has_auto_lending}}
+		{{^has_auto_lending}}<p>NO auto-lending</p>{{/has_auto_lending}}
+
```

Comment:
Is all this stuff just used while developing to view what values you have set?

Reply:
@BoulderBrains - yes - these are placeholders to be replaced with conditional content when one of us gets to that ticket.